main_routers/workshop_router.py

```
@@ -2894,19 +2984,56 @@ async def sync_workshop_character_cards() -> dict:
+                                    'character_origin',
+                                    'display_name',
+                                    origin_display_name,
+                                )
+                                set_reserved(
+                                    catgirl_data,
+                                    'character_origin',
+                                    'model_ref',
+                                    model_binding.get('model_ref', ''),
+                                )
+
+                            # 如果角色卡带有可识别的模型路径，同时保存当前 avatar 绑定信息
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)